ftdetect/set_repl_cmd.vim

```diff
@@ -1,117 +1,115 @@
-    au FileType elixir
-          \ if filereadable('config/config.exs') |
-          \   call neoterm#repl#set('iex -S mix') |
-          \ elseif &filetype == 'elixir' |
-          \   call neoterm#repl#set('iex') |
-          \ endif
-    " Julia
-    au FileType julia
-          \ if executable('julia') |
-          \   call neoterm#repl#set('julia') |
-          \ end
-    " PARI/GP
-    au FileType gp
-          \ if executable('gp') |
-          \   call neoterm#repl#set('gp') |
-          \ end
-    " R
-    au FileType r,rmd
-          \ if executable('R') |
-          \   call neoterm#repl#set('R') |
-          \ end
-    " Octave
-    au FileType octave
-          \ if executable('octave') |
-          \   if executable('octave-cli') |
-          \     if g:neoterm_repl_octave_qt |
-          \       call neoterm#repl#set('octave --no-gui') |
-          \     else |
-          \       call neoterm#repl#set('octave-cli') |
-          \     end |
-          \   else |
-          \     call neoterm#repl#set('octave') |
-          \   end |
-          \ end
-    " MATLAB
-    au FileType matlab
-          \ if executable('matlab') |
-          \   call neoterm#repl#set('matlab -nodesktop -nosplash') |
-          \ end
-    " Idris
-    au FileType idris,lidris
-          \ if executable('idris') |
-          \   call neoterm#repl#set('idris') |
-          \ end
-    " Haskell
-    au FileType haskell
-          \ if executable('stack') |
-          \ call neoterm#repl#set('stack ghci') |
-          \ elseif executable('ghci') |
-          \   call neoterm#repl#set('ghci') |
-          \ end
-    au FileType php
-          \ let s:argList = split(g:neoterm_repl_php) |
-          \ if len(s:argList) > 0 && executable(s:argList[0]) |
-          \   call neoterm#repl#set(g:neoterm_repl_php) |
-          \ elseif executable('psysh') |
-          \   call neoterm#repl#set('psysh') |
-          \ elseif executable('php') |
-          \   call neoterm#repl#set('php -a') |
-          \ end
-    " Clojure
-    au FileType clojure
-          \ if executable('lein') |
-          \   call neoterm#repl#set('lein repl') |
-          \ end
-    " Lua
-    au FileType lua
-          \ if executable('luap') |
-          \   let s:lua_repl='luap' |
-          \ elseif executable('lua') |
-          \   let s:lua_repl='lua' |
-          \ endif |
-          \ if executable('luarocks') && exists('s:lua_repl') |
-          \   call neoterm#repl#set(s:lua_repl . ' -l"luarocks.require"') |
-          \ endif
-    " TCL
-    au FileType tcl
-          \ if executable('tclsh') |
-          \   call neoterm#repl#set('tclsh') |
-          \ endif
-    " Standard ML (SML)
-    au FileType sml
-          \ if executable('sml') |
-          \   if executable('rlwrap') |
-          \     call neoterm#repl#set('rlwrap sml') |
-          \   else |
-          \     call neoterm#repl#set('sml') |
-          \   endif |
-          \ endif
-  aug END
-end
+aug set_repl_cmd
```

Comment:
I don't know why this file is marked as completely changed. I just removed the `if` to check if the editor is Neovim.

Reply:
It's all marked as changed because of the indentation